packages/bot-skeleton/src/constants/messages.js

```diff
@@ -7,7 +7,7 @@ export const unrecoverable_errors = [
     'NotDefaultCurrency',
     'PleaseAuthenticate',
     'FinancialAssessmentRequired',
-    'PositiveNumberExpected',
+    'PositiveIntegerExpected',
```

Comment:
Decimal numbers are not integers so how does this work?

Reply:
we check the decimal number by using the API and we don't need `PositiveNumberExpected ` at all but I've changed this part because we had another bug which not raised yet. if a user inserts a decimal number for the duration and runs the bot gets the error for the "Positive integer number" but the Bot is not stopped. this change makes but stops in this situation.

Reply:
@nijil-binary  and I just tried giving an arithmetical operation in amount and if we give `1 - 100`, the bot doesn't stop.

Reply:
@carol-binary 
the different message comes from API,  I've updated the code, and now it works fine